lnrpc/lightning.proto

```diff
@@ -2638,6 +2638,12 @@ message QueryRoutesRequest {
     fallback.
     */
     repeated lnrpc.FeatureBit dest_features = 17;
+
+    /*
+    The time preference for this payment. Set to -1 to optimize for fees
+    only, to 1 to optimize for reliability only or a value inbetween for a mix.
+    */
+    double time_pref = 18;
```

Comment:
Only added to `QueryRoutes` for now. Would first like to know what people think of the approach. Later it can be added to `SendPayment` too.

Reply:
Added SendPayment too